backend/prisma/schema.prisma

```diff
@@ -19,10 +19,12 @@ model User {
   username                  String   @unique
   password                  String
   currentHashedRefreshToken String?
```

Comment:
whenever you change something in the `schema.prisma` file, you should run `npm run prisma:migrate:dev name_of_migration` and rerun the application with `npm run start:dev` for the database changes to reflect